src/targz_sizes.c

```diff
@@ -10,12 +11,39 @@
 
 #include <stdint.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <getopt.h>
 
 #include <zlib.h>
 
 #define COMPRESSED_BUFFER 102400
 #define TAR_BLOCK_SIZE 512
 
+#ifndef max
+    #define max( a, b ) ( ((a) > (b)) ? (a) : (b) )
```

Comment:
I'd prefer the `min` and `max` macros to be normal inline static functions; macros were historically used because (1) old compilers didn't know how to inline functions properly (but for at least 15 years now every compiler does that), (2) to make them work for any type with well-defined `<` (but here only `int`s matter).